app/controllers/sdg_management/targets_controller.rb

```diff
@@ -1,6 +1,8 @@
 class SDGManagement::TargetsController < SDGManagement::BaseController
   Target = ::SDG::Target
 
+  load_and_authorize_resource class: "SDG::Target"
+
   def index
     @targets = Target.all.sort
```

Comment:
`@targets = @targets.sort`. This way we only use the targets which were loaded by the `load_and_authorize_resource` method.

And now the `Target = ::SDG::Target` serves no purpose AFAIK, so we can remove it.